.gitignore

```diff
@@ -5,3 +5,4 @@ site
 .sass-cache
 .publish
 ___test-screenshots___
+yarn.lock
```

Comment:
yarn.lock should be in the repo:

> Yarn will generate a yarn.lock file within the root directory of your package. You don’t need to read or understand this file - just check it into source control.

https://yarnpkg.com/en/docs/migrating-from-npm